base_search_fuzzy/models/trgm_index.py

```diff
@@ -41,6 +42,7 @@ class TrgmIndex(models.Model):
         string="Index Type",
         default="gin",
         required=True,
+        ondelete={"gin": "set default", "gist": "set default"},
         help='Cite from PostgreSQL documentation: "As a rule of thumb, a '
```

Comment:
This help message is a bit old, pg9.6 old. The performance tradeoff in maintaining GIN indexes in modern postgres is negligible. In fact it is removed from documentation. Maybe we are better to just reference the docs here.

https://www.postgresql.org/docs/current/textsearch-indexes.html